### PR TITLE
Pt/contact info

### PR DESCRIPTION
**Pull request by paaaatrick**
## Trello Card(s)
https://trello.com/c/RIbiIuKP/67-section-contact-info

## What was updated?
- adds template and styles for contact info section, using fire site settings like in footer, and hardcodes the fax content. 
- adds icons to assets

## Screenshots of updates (visual changes)

Mobile
Screenshot 2020-01-08 10 56 05

Desktop
Screenshot 2020-01-08 10 56 17

### CODE REVIEW

File: config/field.field.node.page.field_sections.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ dependencies:
     - node.type.page
     - paragraphs.paragraphs_type.banner_with_links
     - paragraphs.paragraphs_type.callout_grid
-    - paragraphs.paragraphs_type.callout_with_file_download
```
Comment (1forh) [inline config/field.field.node.page.field_sections.yml]: this shouldn't be removed. must have happened when fixing merge conlicts

File: web/themes/fire/templates/components/contact-info/paragraph--contact-info.html.twig [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      <div class="contact-info__item">
+        <div class="contact-info__item-icon">
+        {% include directory ~ "/assets/media/svgs/icon--fax.svg" %}
+        </div>
+
+         <h1 class="contact-info__item-title">Fax</h1>
+
+        <div class="contact-info__item-description">
+          <p>
+            <span>330-477-0626</span> <br/>
```
Comment (1forh) [inline web/themes/fire/templates/components/contact-info/paragraph--contact-info.html.twig]: let's add these to the site settings form.

File: config/field.field.node.page.field_sections.yml (hunk 1 above)
Comment (paaaatrick) [reply config/field.field.node.page.field_sections.yml]: oops, added back now.

File: web/themes/fire/templates/components/contact-info/paragraph--contact-info.html.twig (hunk 2 above)
Comment (paaaatrick) [reply web/themes/fire/templates/components/contact-info/paragraph--contact-info.html.twig]: ty for showing me how to do this.

**Pull request merged**